src/components/app-content-renderer/second-panel.js

```diff
@@ -11,9 +10,7 @@ const SecondPanel = () => {
 
   return (
     <div className="second-panel">
-      <Grid className="pf-u-p-lg pf-u-pt-xl pf-u-pb-xl" hasGutter>
-        {recommendationRenderer(recommendations)}
-      </Grid>
+      <div className="container">{recommendationRenderer(recommendations)}</div>
```

Comment:
This classname is too vague, can you change it to something more reasonable? Like `ins-c-recommendation__container`